zerver/models/messages.py

```diff
@@ -622,9 +622,7 @@ def get_usermessage_by_message_id(
     user_profile: UserProfile, message_id: int
 ) -> Optional[UserMessage]:
     try:
-        return UserMessage.objects.select_related().get(
-            user_profile=user_profile, message_id=message_id
-        )
+        return UserMessage.objects.get(user_profile=user_profile, message_id=message_id)
```

Comment:
This first commit should have a commit message explaining why it's desirable and correct to remove this -- I know it's in the PR description, but it's the commits that are most handy for posterity.